src/Computer.php

```diff
@@ -222,6 +217,10 @@ public function post_updateItem($history = 1)
                      $item->getFromDB($tID);
                     if (!$item->getField('is_global')) {
                         $changes['id'] = $item->getField('id');
+                        //propage is_dynamic value if needed to prevent locked fields
+                        if (isset($item->fields['is_dynamic'])) {
+                            $changes['is_dynamic'] = $item->fields['is_dynamic'];
```

Comment:
IMHO, this is no correct, as you will also apply the `is_dynamic=1` to all following items in the loop, even if they are not dynamic.